code-challenges.js

```diff
@@ -13,6 +24,15 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+const odd = (arr) =>{
+  for (let i=0; i<arr.length; i++){
+    var newArr = arr.filter(arr =>  arr % 2 !== 0 && typeof arr === "number")
+    return newArr.sort(function(a, b){return a-b});
+
+  }
```

Comment:
you can take away the for loop in this solution and it still works exactly the same since you're using a filter :)